linfa-linear/src/glm/distribution.rs

```diff
@@ -4,6 +4,8 @@ use ndarray::Zip;
 
 use crate::error::{LinearError, Result};
 
+const NEG_INFINITY: f64 = -1.0_f64 / 0.0_f64;
```

Comment:
I believe we should be using `std::ft64::NEG_INFINITY` instead (for compatibility with Rust 1.38). Better than defining our own constant.

Reply:
(it's deprecated in favor of f64::NEG_INFINITY in new rust but not available in 1.38, so since we want to be backwards compatible we need to use that instead).

Reply:
Thanks, I have made the change.